koi.gemspec

```diff
@@ -15,21 +15,17 @@ Gem::Specification.new do |s|
 
   s.files         = `git ls-files`.split("\n")
   s.test_files    = `git ls-files -- {test,spec,features}/*`.split("\n")
-  s.executables   = `git ls-files -- bin/*`.split("\n").map{ |f| File.basename(f) }
   s.require_paths = ["lib"]
 
   # MVC Framework
-  s.add_dependency 'rails'                         , '~> 3.2.16'
+  s.add_dependency 'rails'                         , '~> 3.2.0'
 
   # Overwrite for default rails
   s.add_dependency 'jquery-rails'                  , '~> 3.1.1'
 
   # jQuery UI
   s.add_dependency 'jquery-ui-rails'               , '~> 4.2.1'
 
-  # Database
-  s.add_dependency 'mysql2'                        , '~> 0.3.11'
```

Comment:
Does this mean we could potentially use postgres in V1 projects now, assuming there isn't any custom sql in any of koi that would break it?

Reply:
Possibly although I haven't tested that. The major reason for this change is to give the user more flexibility about how they configure their database.